openlibrary/templates/account/readinglog_stats.html

```diff
@@ -104,6 +104,21 @@
         &raquo;
     </div>
     <h1>$_('My %(shelf_name)s Stats', shelf_name=shelf_name)</h1>
+    $code:
+        mb=MyBooksTemplate(username=owner_key.split('/')[-1] ,key='mybooks')
+    $if shelf_key == 'already-read' and yearly_reads:
+    <div class="breadcrumb-wrapper">
+        <div class="sansserif grey account-settings-menu navigation-breadcrumbs">
+            $ all_url = '/people/openlibrary/books/already-read/stats'
+            $ year_url = '/people/openlibrary/books/already-read/stats/%d'
```

Comment:
We don't want this pinned to the `/people/openlibrary` ; it needs to be the `owner_key`
```suggestion
            $ all_url = owner_key + '/books/already-read/stats'
            $ year_url = owner_key + '/books/already-read/stats/%d'
```